### PR TITLE
Add function show activity

### PR DESCRIPTION
**Pull request by chintk**

### CODE REVIEW

File: app/models/enrollment_subject.rb [hunk 1]
```diff
@@ -2,4 +2,15 @@ class EnrollmentSubject < ActiveRecord::Base
   belongs_to :course
   belongs_to :subject
   belongs_to :user
+
+  after_update :insert_activity
+
+  private
+  def insert_activity
+    if self.status && !self.user.activities.find_by(
```
Comment (dieunb) [inline app/models/enrollment_subject.rb]: should set a new variable for below code and remembering remove () 
```
self.user.activities.find_by(acti: 2, course: self.course, subject: self.subject)
```

File: app/models/enrollment_task.rb [hunk 2]
```diff
@@ -3,4 +3,16 @@ class EnrollmentTask < ActiveRecord::Base
   belongs_to :task
   belongs_to :subject
   belongs_to :course
+
+  after_update :insert_activity
+
+  private
+  def insert_activity
+    if self.status && !self.user.activities.find_by(
```
Comment (dieunb) [inline app/models/enrollment_task.rb]: follow above comment

File: app/models/enrollment_task.rb [hunk 3]
```diff
@@ -3,4 +3,16 @@ class EnrollmentTask < ActiveRecord::Base
...
+      acti: 2, course: self.course, subject: self.subject, task: self.task)
+      @activity = self.user.activities.build(
```
Comment (dieunb) [inline app/models/enrollment_task.rb]: remove ()

File: app/models/user.rb [hunk 4]
```diff
@@ -16,7 +17,7 @@ class User < ActiveRecord::Base
   has_secure_password
   validates :password, length: {minimum: 6}, allow_blank: true
 
-  scope :normal, ->{where(suppervisor: false)}
+  scope :normal, ->{where(suppervisor: !true)}
```
Comment (dieunb) [inline app/models/user.rb]: why changing?
```
suppervisor: false
```

Comment (chintk) [inline app/models/user.rb]: because suppervisor may be null

File: app/models/enrollment_task.rb (hunk 3 above)
Comment (chintk) [inline app/models/enrollment_task.rb]: if remove(), line may be too long?

File: app/models/enrollment_task.rb [hunk 5]
```diff
@@ -3,4 +3,14 @@ class EnrollmentTask < ActiveRecord::Base
   belongs_to :task
   belongs_to :subject
   belongs_to :course
+
+  after_update :insert_activity
+
+  private
+  def insert_activity
+    if self.status
+      @activity = self.user.activities.build acti: 2, course: self.course, subject: self.subject, task: self.task
```
Comment (dieunb) [inline app/models/enrollment_task.rb]: remove self

**Pull request merged**